### PR TITLE
add script to increment helm charts version

### PR DESCRIPTION
**Pull request by plaharanne**
## Summary of changes

https://github.com/crate/cloud/issues/614

## Checklist

- [x] Relevant changes are reflected in `CHANGES.rst`
- [ ] Added or changed code is covered by tests
- [x] Documentation has been updated if necessary
- [x] Changed code does not contain any breaking changes (or this is a major version change)

### CODE REVIEW

File: deploy/charts/crate-operator/README.md [hunk 1]
```diff
@@ -21,10 +21,18 @@ Once Helm is properly set up, install the chart.
...
+```
 helm install crate-operator crate-operator
 ```
 
+### Install from repo
+
+```
+helm repo add crate-operator https://crate.github.io/crate-operator
+helm search repo crate-operator
+helm install crate-operator/crate-operator
```
Comment (SStorm) [inline deploy/charts/crate-operator/README.md:33]: I believe you also have to specify the name of the distribution you are installing? At least this simple install does not work for me:
```
$ helm install crate-operator/crate-operator
Error: INSTALLATION FAILED: must either provide a name or specify --generate-name

$ helm install my-crate-operator crate-operator/crate-operator --create-namespace
NAME: my-crate-operator
LAST DEPLOYED: Thu Sep 22 07:39:20 2022
NAMESPACE: crate-operator
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Thank you for installing crate-operator.


```

**Review by SStorm — APPROVED** [review]
Nice extensive work on this, thank you Pauline!

File: deploy/charts/crate-operator/README.md (hunk 1 above)
Comment (SStorm) [reply deploy/charts/crate-operator/README.md:33]: FWIW
```
$ helm version
version.BuildInfo{Version:"v3.7.0", GitCommit:"eeac83883cb4014fe60267ec6373570374ce770b", GitTreeState:"clean", GoVersion:"go1.16.8"}

```

Comment (plaharanne) [reply deploy/charts/crate-operator/README.md:33]: you're right, it should be `helm install crate-operator crate-operator/crate-operator`

**Pull request merged**